models/articles.model.js

```diff
@@ -4,13 +4,25 @@ exports.selectArticleById = (id) => {
   return db
     .query(`SELECT * FROM articles WHERE article_id=$1;`, [id])
     .then(({ rows }) => {
-        if(rows.length === 0 ){
-            return Promise.reject({
-                status: 404,
-                msg: "No article found by that ID"
-            })
-        }
+      if (rows.length === 0) {
+        return Promise.reject({
+          status: 404,
+          msg: "No article found by that ID",
+        });
+      }
       return rows;
-    })
+    });
+};
+
+exports.updateArticleById = (id, updates) => {
+  let newVotes = updates.inc_votes;
```

Comment:
I'd quite like it if this was taken off the body either up in the controller where you've got req.params.article_id OR bring the req.params one down into the model alongside this one.